packages/warriorjs-core/src/Unit.js

```diff
@@ -92,10 +92,16 @@ class Unit {
    */
   performTurn() {
     if (this.isAlive()) {
```

Comment:
This check is now redundant with the inclusion of the inner `.isAlive()` checks.